src/main/java/gamer/ComGamer.java

```diff
@@ -0,0 +1,62 @@
+/**
+* @Package_name   : gamer
+* @Class_name     : ComGamer 
+*
+* Create Date : 2023-10-21 
+* Create User : 정은채
+*/
+package gamer;
+
+import java.util.ArrayList;
+
+import utility.Utility;
+
+public class ComGamer implements Gamer{
+	//생성자 
+	public ComGamer() {
+		this.comNumberArr = new ArrayList<>();
+	}
+	
+	
+	/**
+	* description : ComGamer의 게임 플레이
+	*
+    * @Method : play
+    */ 
```

Comment:
```suggestion
	/**
	* description : ComGamer의 게임 플레이
	*
        * @Method : play
        */ 
```

@songjayhyun 님께서 말씀주신 대로 인덴트 설정 참고하시면 좋을 듯 합니다:)

Reply:
앗ㅜㅜ 이렇게 된 게 있었군요ㅠ